compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -230,10 +230,8 @@ getVectorPreProcStrategy(linalg::LinalgOp linalgOp) {
     if (hasAnySVEFeature(targetAttr)) {
       return VectorPreProcStrategy::Masking;
     }
-    if ((linalg::isElementwise(linalgOp) || isFullyDynamicOp(linalgOp)) &&
-        enableVectorPeeling) {
-      return VectorPreProcStrategy::Peeling;
-    }
+
+    return VectorPreProcStrategy::Peeling;
```

Comment:
I think we still need to check if `enableVectorPeeling` is true or not. Otherwise, users will be confused if they explicitly disable peeling through flags.

Reply:
Good catch, thanks!